sections/about-us/about-us.module.scss

```diff
@@ -24,6 +23,10 @@
       font-weight: 400;
     }
 
-    
   }
 }
+
+.about-us-container {
```

Comment:
This probably needs `@include page-margin;` (and an additional <div> to apply the `width: 80%` within the page-margin container)